packages/terra-table/src/proptypes/columnShape.js

```diff
@@ -42,7 +47,19 @@ const columnShape = PropTypes.shape({
    * One of `ascending`, `descending`.
    */
   sortIndicator: PropTypes.oneOf(Object.values(SortIndicators)),
+
```

Comment:
If we are going to have the highlighting at the column shape level, we should create an isHighlighted prop.

Reply:
We can avoid this additional prop by directly checking if colour is provided for a column or not